src/views/Home/Home.jsx

```diff
@@ -13,7 +13,46 @@ function Home() {
 	*/
 	return (
 		<div className={styles.wrapper}>
-			<p>Home</p>
+			<div className={styles.contentBoxOne}>
+				<h1> Welcome to the brains</h1>
+				<h3>
+					Welcome to ask the brains, where we answer your questions to
+					sell you the perfect virtual pet
+				</h3>
+			</div>
+			<div className={styles.primaryContentContainer}>
+				<div className={styles.contentBoxTwo}>
+					<h2>We do AI</h2>
+					<h2 className={styles.purpleH2}>(via ChatGPT)</h2>
+					<p>
+						Ask us a question, we'll secretly pass it on to ChatGPT
+						and will return the result pretending that we generated
+						it ourselves
+					</p>
+				</div>
+				<div className={styles.contentBoxThree}>
+					<img
+						className={styles.homeIll}
+						src="public/img/home_ill.png"
```

Comment:
Sorry, my bad 😅 . This should be "/img/home_ill.png" without the public bit.

Reply:
I was able to test it using DevTools on the site so 🤞

Reply:
right: have corrected
`src="/img/home_ill.png"`